spec/abstract-operations.html

```diff
@@ -873,6 +873,32 @@ <h1>
   <emu-clause number="5" id="sec-operations-on-records">
     <h1>Operations on Records</h1>
 
+    <emu-clause id="sec-recordtoobject" type="abstract operation">
+      <h1>
+        <ins>
+          RecordToObject (
+            _value_: a Record,
+          ): an Object
+        </ins>
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It is used to specify the creation of objects based on the contents of a record.</dd>
+      </dl>
+      <emu-alg>
+          1. Let _obj_ be OrdinaryObjectCreate(*null*, &laquo; [[RecordData]] &raquo;).
+          1. Set _obj_.[[RecordData]] to _value_.
```

Comment:
an object with a slot is exotic, isn't it?

Reply:
nope. ordinary and exotic are only about the essential-internal-methods. https://tc39.es/ecma262/multipage/overview.html#sec-exotic-object The

Reply:
ah, fair enough

Reply:
TBF I only recently learnt this as part of spec reading related to thinking about this issue 😃